.drone.yml

```diff
@@ -0,0 +1,227 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/password_policy
+
+branches: [master, release*, release/*]
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    version: ${OC_VERSION}
+    pull: true
+    db_type: ${DB_TYPE}
+    db_name: ${DB_NAME}
+    db_host: ${DB_TYPE}
+    db_username: autotest
+    db_password: owncloud
+    exclude: apps/password_policy
```

Comment:
Why the exclude? this is not required for general apps ( only the ones that are shipped in the qa-tarball which is market / notifications / encryption ). It doesn't hurt - but is not required and might confuse people when copy+pasting it around

Reply:
I copy-pasted it from somewhere! So exactly this problem, I don't really know if it is needed.